backend/authschemes/localauth/local_auth.go

```diff
@@ -77,13 +78,13 @@ func (LocalAuthScheme) Type() string {
 //
 // In each case above, the actual action is deferred to the bridge connecting this auth scheme to
 // the underlying system/database
-func (p LocalAuthScheme) BindRoutes(r *mux.Router, bridge authschemes.AShirtAuthBridge) {
+func (p LocalAuthScheme) BindRoutes(r chi.Router, bridge authschemes.AShirtAuthBridge) {
 	remux.Route(r, "POST", "/register", remux.JSONHandler(func(r *http.Request) (interface{}, error) {
 		if !p.RegistrationEnabled {
 			return nil, fmt.Errorf("registration is closed to users")
 		}
 
-		dr := remux.DissectJSONRequest(r)
+		dr := dissectors.DissectJSONRequest(r)
```

Comment:
I'll put more info below, but the remux dissector functions aren't helpful anymore now that we're not using gorilla mux, so now the code just calls the dissectors directly

Reply:
no longer relevant